xen-posix/src/mini_libc.c

```diff
@@ -321,4 +323,10 @@ unsupported_function_crash(read);
 unsupported_function_crash(gmtime);
 unsupported_function_crash(strtod);
 unsupported_function_crash(rename);
-unsupported_function_crash(strerror);
+
+unsupported_function_log(size_t, fwrite, 1);
+unsupported_function_log(int, fputc, 1);
+unsupported_function_log(int, fputs, 1);
```

Comment:
These return codes look wrong. But why do we need these functions anyway? Presumably they're never used, or linking would have failed before now.